app/assets/stylesheets/style.scss

```diff
@@ -328,4 +350,45 @@ label:hover {
   }
 }
 
+#footer {
+position: fixed;
+background-color: #F3F3F3;
+bottom: 0;
+margin-top: 40px;
+width: 100%;
+border-top: 1px solid #D8D8D8;
+height: 50px; /* choose any height */
+  li {
+    list-style: none;
+    display: inline;
+    padding: 0 27px 0 27px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 27px` instead of `0 27px 0 27px`